user/src/com/google/gwt/user/client/rpc/core/java/util/LinkedHashMap_CustomFieldSerializer.java

```diff
@@ -105,15 +108,30 @@ private static boolean getAccessOrder(LinkedHashMap instance) {
     if (!reflectionHasFailed.get()) {
       try {
         Field f = accessOrderField.get();
-        if (f == null || !f.isAccessible()) {
+        boolean isAccessible = f != null && f.isAccessible();
+        if (!isAccessible) {
           f = LinkedHashMap.class.getDeclaredField("accessOrder");
           synchronized (f) {
-            // Ensure all threads can see the accessibility.
-            f.setAccessible(true);
+            try {
+              // see if we can *try* setting the accessOrder field accessible:
+              final Method trySetAccessible = AccessibleObject.class.getDeclaredMethod("trySetAccessible");
```

Comment:
Please wrap this line, as it adds a new checkstyle warning (see the diff tab for where it is logged in this review

Reply:
Thanks for pointing this out. The GWT formatter correctly breaks this line. cad8ec716495d3740fd7dbfea9fbbeef9e644c7d